### DISCUSSION TITLE
Thank you for continuing the work

### DISCUSSION THREAD

**Discussion by The-Code-Monkey** [ghiscoding/lerna-lite#99] [Q&A] [answered]
Lerna lite is a lifesaver and i will be adding it to my current tech stack.

I hope you know how much you are probably helping people out.

I want to add a quick question, Is it possible to run the lerna-lite repo using yarn install instead of npm install

**Comment by ghiscoding** [comment, marked as answer]
The main difference with Lerna-Lite compare to Lerna is that it's only including 3 commands out of the 15 commands that Lerna has. The `bootstrap` command is not included and this mean that you have to setup your own workspace (NPM or Yarn Workspace) prior to using Lerna-Lite. Typically if you come from Lerna and your project was already setup with Yarn Workspace then it's just a matter of replacing Lerna with Lerna-Lite and you're good to go, if however you were using Lerna with `bootstrap` and without NPM or Yarn Workspace then you will need to change your setup to have the workspace setup first.

If you want an example of a Yarn Workspace, you can take a look at my other project [Slickgrid-Universal](https://github.com/ghiscoding/slickgrid-universal) which is setup with Yarn Workspace (yarn 1.x classic to be more specific) and Lerna-Lite, the way I created my own project was to watch couple of YouTube videos (like a lot of people), for example these ones
- [Monorepo: lerna + yarn workspaces](https://www.youtube.com/watch?v=tyYnZSdF3eY) 
- [Monorepos: Yarn Workspaces and Lerna for beginners!](https://www.youtube.com/watch?v=hNJROosGexA)

just make sure to use Lerna-Lite instead of Lerna and the CLI commands are called the same way, but again Lerna-Lite only has 3 commands in total.

Note: I converted your issue to a Discussion because that wasn't an issue with the lib ;)

**Reply by The-Code-Monkey** [reply]
Ah yeah no worries, I kinda need to use the lerna link command but it doesn't seem to exist is there a way to do that with your lib or is that one of the commands you removed.

**Reply by The-Code-Monkey** [reply]
As i currently have a mono repo that requires the link command

**Reply by ghiscoding** [reply]
Like I said earlier, there's only 3 commands and `link` is also not part of that list, but again if you setup an NPM/Yarn/other... Workspace then it will do all the symlink for you, so that command is not needed if you setup a Workspace and I will not add it either. For example Yarn Workspace does the symlinks for you and it's always great to use the hoisting as well which will creates only 1 `node_module` folder in the root of the project. You should take a look at the [Yarn Workspace](https://classic.yarnpkg.com/lang/en/docs/workspaces/) docs

**Reply by The-Code-Monkey** [reply]
ah ok cheers for your help i will close this now.
